### PR TITLE
docs(cli/dts): fix `Deno.startTls` example

### PR DESCRIPTION
**Pull request by caspervonb**

### CODE REVIEW

File: cli/dts/lib.deno.unstable.d.ts [hunk 1]
```diff
@@ -982,7 +982,7 @@ declare namespace Deno {
    *
    * ```ts
    * const conn = await Deno.connect({ port: 80, hostname: "127.0.0.1" });
-   * const tlsConn = await Deno.startTls(conn, { certFile: "./certs/my_custom_root_CA.pem", hostname: "127.0.0.1", port: 80 });
```
Comment (piscisaureus) [inline cli/dts/lib.deno.unstable.d.ts:985]: "127.0.0.1" is not actually a valid hostname. Maybe change that to "localhost"?

**Comment by satyarohith** [comment]
Related https://github.com/denoland/deno/issues/6238

File: cli/dts/lib.deno.unstable.d.ts [hunk 2]
```diff
@@ -982,7 +982,7 @@ declare namespace Deno {
...
+   * const tlsConn = await Deno.startTls(conn, { certFile: "./certs/my_custom_root_CA.pem", hostname: "127.0.0.1" });
```
Comment (caspervonb) [inline cli/dts/lib.deno.unstable.d.ts:985]:
```suggestion
   * const tlsConn = await Deno.startTls(conn, { certFile: "./certs/my_custom_root_CA.pem", hostname: "localhost" });
```

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**